src/panels/lovelace/cards/hui-error-card.ts

```diff
@@ -58,6 +58,8 @@ export class HuiErrorCard extends LitElement implements LovelaceCard {
         color: white;
         padding: 8px;
         font-weight: 500;
+        user-select: text;
+        cursor: default;
```

Comment:
I'm not sure if I follow. Why would we not make it a cursor?

Reply:
Because of consistency.

If being able to select the text was important, then sure. But A: it isn't, and B: if people want to do it, they will click and drag regardless of what the cursor looks like.

![Skärminspelning 2019-07-02 kl  23 56 22 mov](https://user-images.githubusercontent.com/1299821/60549422-19942200-9d25-11e9-8715-bbd415ea25f0.gif)

Reply:
Convinced! 👍